client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -299,25 +310,36 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 					closeModal={ closeUpgradeModal }
 					checkout={ goToCheckout }
 				/>
-				<WebPreview
-					showPreview
-					showClose={ false }
-					showEdit={ false }
-					externalUrl={ siteSlug }
-					showExternal={ true }
-					previewUrl={ previewUrl }
-					loadingMessage={ translate(
-						'{{strong}}One moment, please…{{/strong}} loading your site.',
-						{
-							components: { strong: <strong /> },
-						}
-					) }
-					toolbarComponent={ PreviewToolbar }
-					siteId={ site?.ID }
-					url={ site?.URL }
-					translate={ translate }
-					recordTracksEvent={ recordTracksEvent }
-				/>
+				{ isEnabledStyleSelection ? (
+					<AsyncLoad
+						require="@automattic/design-preview"
+						placeholder={ null }
+						title={ designTitle }
+						description={ selectedDesignDetails?.description }
+						variations={ selectedDesignDetails?.style_variations }
+						isLoading={ isLoadingSelectedDesignDetails }
```

Comment:
Can we unset this `isLoading` prop? This will make the entire theme preview loading to block until the endpoint call finishes. By removing this, at least we can show the title and starts loading the site preview as soon as possible, making the entire experience smoother.

Reply:
Related to this -- maybe we should send the theme description in the `/wpcom/v2/starter-designs` endpoint instead (/ as well)?

This way, as soon as the user clicks on a theme grid cell, we can immediately show the description, and only wait for the style selectors and the site preview to load, making the experience even smoother! What do you think?

Reply:
Both are good ideas, let's update this UX.

Reply:
Updated on my end as well 👍